airbyte-webapp/jest.config.ts

```diff
@@ -13,7 +13,8 @@ const jestConfig: Config = {
   moduleDirectories: ["node_modules", "src"],
   moduleNameMapper: {
     "\\.module\\.scss$": "test-utils/mock-data/mockIdentity.js",
-    "\\.(css|png|svg|scss)$": "test-utils/mock-data/mockEmpty.js",
+    "\\.(css|png|scss)$": "test-utils/mock-data/mockEmpty.js",
+    "\\.svg": "test-utils/mock-data/mockSvg.js",
```

Comment:
The old method wasn't working with svgs that we import using `import { ReactComponent as...`

Solved via the recommended solution from SVGR: https://react-svgr.com/docs/jest/

Reply:
Minor nitpick:

```suggestion
    "\\.(css|png|scss)$": "test-utils/mock-data/mockEmpty.js",
    "\\.svg$": "test-utils/mock-data/mockSvg.js",
```